src/components/jogging/JoggingCartesianAxisControl.tsx

```diff
@@ -17,6 +17,8 @@ type JoggingCartesianAxisControlProps = {
   startJogging: (direction: Direction) => void
   stopJogging: () => void
   disabled?: boolean
+  renderIncrementPressed?: boolean
```

Comment:
Probably want to add a /** foo */ style comment here so it gets a description for intellisense